docs/intro/clients.rst

```diff
@@ -0,0 +1,391 @@
+- *Executing queries.* A  ``Client`` will provide some methods for executing
+  queries against your database. Under the hood, this query is executed using
+  EdgeDB's efficient binary protocol.
+
+.. note::
+
+  For some use cases, you may not need a client library. EdgeDB allows you to
+  execute :ref:`queries over HTTP <ref_edgeql_http>`. This is slower than the
+  binary protocol and lacks support for transactions and rich data types, but
+  may be suitable if a client library isn't available for your language of
+  choice.
+
+Available libraries
+-------------------
+
+To execute queries from your application code, use one of EdgeDB's *client
+libraries* for the following languages.
+
+
+- `JavaScript/TypeScript <https://github.com/edgedb/edgedb-js>`_
+- `Go <https://github.com/edgedb/edgedb-go>`_
+- `Python <https://github.com/edgedb/edgedb-python>`_
+- `Rust <https://github.com/edgedb/edgedb-rust>`_
+
+Unofficial (community-maintained) libraries are available for the following
+languages.
+
+- `.NET <https://github.com/quinchs/edgedb-dotnet>`_
+- `Elixir <https://github.com/nsidnev/edgedb-elixir>`_
+
+
+Usage
+-----
+
+To follow along with the guide below, first create a new directory and
+initialize a project.
+
+.. code-block:: bash
+
+  $ mydir myproject
+  $ cd myproject
+  $ edgedb project init
+
+Configure the environment as needed for your preferred language.
+
+.. tabs::
+
+  .. code-tab:: bash#Node.js
+
+    $ npm init -y
+    $ tsc --init # (TypeScript only)
+    $ touch index.ts
+
+  .. code-tab:: bash#Deno
+
+    $ touch index.ts
+
+  .. code-tab:: bash#Python
+
+    $ python -m venv venv
+    $ source venv/bin/activate
+    $ touch main.py
+
+  .. code-tab:: bash#Rust
+
+    $ cargo init
+
+  .. code-tab:: bash#Go
+
+    $ go mod init example/quickstart
+    $ touch hello.go
+
+  ..   code-tab:: bash#.NET
+
+    $ dotnet new console -o . -f net6.0
+
+Install the EdgeDB client library.
+
+.. tabs::
+
+  .. code-tab:: bash#Node.js
+
+    $ npm install edgedb    # npm
+    $ yarn add edgedb       # yarn
+
+  .. code-tab:: txt#Deno
+
+    n/a
+
+  .. code-tab:: bash#Python
+
+    $ pip install edgedb
+
+  .. code-tab:: toml#Rust
+
+    # Cargo.toml
+
+    [dependencies]
+    edgedb-tokio = "0.3.0"
+    # additional dependencies
+    tokio = { version = "1", features = ["full"] }
+    anyhow = "1.0.63"
+
+  .. code-tab:: bash#Go
+
+    $ go get github.com/edgedb/edgedb-go
+
+  .. code-tab:: bash#.NET
+
+    $ dotnet add package EdgeDB.Net.Driver
+
+Copy and paste the following simple script. This script initializes a
+``Client`` instance. Clients manage an internal pool of connections to your
+database and provide a set of methods for executing queries.
+
+.. note::
+
+  Note that we aren't passing connection information (say, a connection
+  URL) when creating a client. The client libraries can detect that
+  they are inside a project directory and connect to the project-linked
+  instance automatically. (More on this later.)
```

Comment:
What does `(More on this later.)` mean? Is it referenced elsewhere in the documentation? If so, we should hyperlink "project-linked instance automatically." If not, will we get this information in the future through something like a blog post? Unsure what this is meant to entail.

Reply:
Added a ref to the "Connection" section later in the document (which is what I was referring to here). Good call.